Assets/Mirror/Runtime/NetworkAuthenticator.cs

```diff
@@ -59,6 +69,20 @@ public abstract class NetworkAuthenticator : MonoBehaviour
         /// <param name="conn">Connection of the client.</param>
         public abstract void OnClientAuthenticate(NetworkConnection conn);
 
+        protected void ClientAccept(NetworkConnection conn)
+        {
+            OnClientAuthenticated.Invoke(conn);
+        }
+
+        protected void ClientReject(NetworkConnection conn)
+		{
```

Comment:
```suggestion
        {
```
spaces instead of tabs

Reply:
Fixed via commit